google-cloud-logging/src/main/java/com/google/cloud/logging/LogEntry.java

```diff
@@ -303,6 +354,23 @@ public Operation getOperation() {
 
 
   /**
+   * Returns the resource name of the trace associated with the log entry, if any. If it contains a
+   * relative resource name, the name is assumed to be relative to `//tracing.googleapis.com`.
+   */
+  public String getTrace() {
+    return trace;
+  }
+
+
+  /**
+   * Returns the source code location information associated with the log entry if any.
```

Comment:
I'm not the English language expert, but the `getTrace()` method doc has comma before "if any", and this one does not.